src/CommitmentsV2/SFA.DAS.CommitmentsV2/DependencyResolution/CurrentDateTimeRegistry.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using SFA.DAS.CommitmentsV2.Configuration;
+using SFA.DAS.CommitmentsV2.Domain.Interfaces;
+using SFA.DAS.CommitmentsV2.Services;
+using StructureMap;
+
+namespace SFA.DAS.CommitmentsV2.DependencyResolution
+{
+    public class CurrentDateTimeRegistry : Registry
+    {
+        public CurrentDateTimeRegistry()
+        {
+            For<ICurrentDateTime>().Use<CurrentDateTime>()
+                .Ctor<DateTime?>("value").Is(x => x.GetInstance<CommitmentsV2Configuration>().CurrentDateTime);
+        }
```

Comment:
This is from CV-49 branch - just checked and it's still in CV-109. Looking at the code so long as the configured value is not set then this is the correct behaviour. i.e. it will use the system UTC date time.